src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.cs

```diff
@@ -14,6 +18,58 @@ internal static void VerifyValidPath(string path, string argName)
             }
         }
 
+        internal static bool Copy(string sourcePath, string destinationPath, bool recursive,
```

Comment:
nit: rename this method to `CopyDirectory`, and add `full` int the argument names to indicate the caller must provide the method with full paths.